csharp/src/Google.Protobuf/JsonFormatter.cs

```diff
@@ -170,6 +183,27 @@ public void Format(IMessage message, TextWriter writer)
         }
 
         /// <summary>
+        /// Formats the specified field of the message as JSON.
+        /// </summary>
+        /// <param name="message">The message to format.</param>
+        /// <param name="field">The field of the message to format.</param>
+        /// <param name="writer">The TextWriter to write the formatted message to.</param>
+        /// <returns>The formatted field.</returns>
+        public void Format(IMessage message, FieldDescriptor field, TextWriter writer)
```

Comment:
I think it's reasonable, because there's not a lot of ambiguity when you pass in a message - it's reasonably obvious that you're formatting "the message". Here I think the more specific name is useful.